codeinterpreterapi/schema/file.py

```diff
@@ -6,6 +6,10 @@ class File(BaseModel):
     name: str
     content: bytes
 
+    @classmethod
+    def from_bytes(cls, name: str, content: bytes):
```

Comment:
Theoretically, this is not needed because you can create a File(name=name, content=content) but if you find this easier it can stay :)